### PR TITLE
Use FullStory SDK for snippet loading

### PR DESCRIPTION
**Pull request by arielsilvestri**
- Uses the snippet initializer provided by FullStory's browser SDK, preventing the need for manual updates on our end when snippet updates.
- Removes locally-managed init-script.

## Testing

- [x] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
  - Modified existing unit test, FullStory loads script tag in slightly differently.
- [ ] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment
  - Plugin still loads with modified code.

### CODE REVIEW

**Comment by pooyaj** [comment]
I'll let @danieljackins review this one

File: packages/destination-actions/src/destinations/cordial/generated-types.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ export interface Settings {
    */
   apiKey: string
   /**
-   * Cordial API endpoint. Leave default, unless you've been provided with another one
+   * Cordial API endpoint. Leave default, unless you've been provided with another one. [See more details](https://support.cordial.com/hc/en-us/sections/200553578-REST-API-Introduction-and-Overview)
```
Comment (danieljackins) [inline packages/destination-actions/src/destinations/cordial/generated-types.ts:9]: Is this change expected?

**Review by danieljackins — APPROVED** [review]
LGTM other than that one (unintended?) change, I'd just do some testing locally and/or on staging to verify its working!

**Comment by pooyaj** [comment]
+1 on testing on stage. Also let's check the bundle sizes to make sure we don't impact the bundle size for other destinations

File: packages/destination-actions/src/destinations/cordial/generated-types.ts (hunk 1 above)
Comment (arielsilvestri) [reply packages/destination-actions/src/destinations/cordial/generated-types.ts:9]: mmm nope, I didn't do that myself.

**Comment by arielsilvestri** [comment]
@pooyaj @danieljackins What's the address for browser destinations on staging - is this just in our segment staging env?

**Pull request merged**